Gruntfile.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * flopmang
+ * https://github.com/dylancwood/flopmang
+ *
+ * Copyright (c) 2014 Dylan Wood
+ * Licensed under the MIT license.
+ */
+
+'use strict';
+
+module.exports = function (grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+        watch: {
+            dev: {
+                files: ['lib/*.js', 'test/*.js'],
+                tasks: ['jshint', 'test']
+            }
+        },
+
+        jshint: {
+            all: [
+                'Gruntfile.js',
+                'lib/*.js',
+                'test/**.js'
+            ],
+            options: {
+                jshintrc: '.jshintrc',
+            },
+        },
+
+        jscs: {
+            src: {
+                options: {
+                    config: '.jscs.json'
+                },
+                files: {
+                    src: [
+                        'Gruntfile.js',
+                        'lib/*.js',
+                        'test/**.js'
+                    ],
+                }
+            },
+        },
+
+        mochacli: {
+            options: {
+                files: 'test/*_test.js'
+            },
+            spec: {
+                options: {
+                    reporter: 'spec',
+                    timeout: 10000
+                }
+            }
+        },
+
+        bump: {
```

Comment:
i have mixed feeling on auto-incrementing version and pushing.  how do we honor semver?  i just dont have experience with this, so maybe it can be explained how it fits in the process